### ISSUE TITLE
Add option to enable sterioscopic

### ISSUE THREAD

**Issue by bnolan** [bnolan/scenevr-web#106]
For people on google cardboard, or other viewers that want to force-enable stereoscopic.

**Comment by bnolan** [comment]
Be also good to add analyglyph (red/blue glasses) support.

**Comment by SBUtltmedia** [comment]
Hi Ben, we've been working on adding stereoscopic by integrating [StereoEffect.js](https://github.com/mrdoob/three.js/blob/master/examples/js/effects/StereoEffect.js) and some code from a cardboard example and would love to submit a pull request soon. How would you suggest we add the option though?

**Comment by bnolan** [comment]
I'd use [the webvr polyfill](https://github.com/borismus/webvr-polyfill). You'll need to add the stereo effect renderer to `tick` in client.js, where `.render()` is called. It'd also be good if you could trigger the `click` event when the magnetic switch is moved on cardboard. Otherwise you could integrate some kind of [gaze tracking](https://github.com/neuman/vreticle). I could add a tag to meshes to say whether or not they are hooked up to recieve click events, in which case you could trigger an even when the gaze is on it.

Basically it'd be good to have some interactivity for cardboard users, and not just passive observers. :)
